Dockerfile.frontend-container

```diff
@@ -1,15 +1,19 @@
-FROM node:18
+
+FROM node:16-alpine
```

Comment:
What is the reason that we are going to an older version of node?

Reply:
Image size is a little less and we don't want the updates on the new release but it's a small difference 847MB for the whole image vs 907. So reverting to node 19.